### PR TITLE
Improvements to documentation

### PR DESCRIPTION
**Pull request by nicoandresr**
Improvements to documentation for use the package with npm.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,51 @@ For some more information about Constrained Local Models, take a look at Xiaogua
...
+  }
+
+  startFaceTracking() {
+    this.canvasContext.drawImage(this.webcam.domElement, 0, 0, 320, 240);
+    this.tracker.draw(this.canvas);
+  }
+}
+```
+
+## From Vainilla Script ##
```
Comment (markknol) [inline README.md]: What is _Vainilla Script_ ?

Comment (nicoandresr) [reply README.md]: sorry, is a typo 🙈 , I want says 'Vanilla Script' or 'Pure Javascript' 😸

**Comment by nicoandresr** [comment]
Please somebody that's review and merge this PR...

**Comment by nicoandresr** [comment]
I think that this project is deserted, my PR is open along 6 months. T_T

**Pull request closed without merging**